modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewTest.java

```diff
@@ -5472,4 +5477,55 @@ public String toString() {
 
         sl.dispose();
     }
+
+    @Test
+    // see JDK-8177945
```

Comment:
Minor: It seems best to not have the comment between the annotation and the method being annotated. I see this pattern also in `VirtualFlowTest`, but you might want to move the comment above the annotation.